app/controllers/maily/emails_controller.rb

```diff
@@ -1,5 +1,7 @@
 module Maily
   class EmailsController < Maily::ApplicationController
+    include Maily::EmailsHelper
```

Comment:
Rails way ™️ will be something like: defining the method in https://github.com/markets/maily/blob/master/app/controllers/maily/application_controller.rb and use the `helper_method` to make it available in the views